[AIP-134] Approve the AIP for Update. (#11)
googleapis/aip: aip/general/0158.md

internal protocol buffer message with any data needed, and send the serialized
proto, base-64 encoded.

> Even with obfuscation protection, the page token might still be modified by a
> caller. The service **must** validate the contents of the page token and authenticate
> the caller for any access to resources specified in the page token.
> 

Reviewer: This is interesting because it implies the page token includes some level of access control (like an OAuth2 access token) -- which it should not. 

Perhaps we can rephrase this to clarify that the page token must be useful only as a cursor and should carry with it no implication of any form of authorization to list or retrieve any of the items involved in a paginated list response. 

```suggestion
Page tokens **must** be limited to providing an indication of where to continue
the pagination process only. They **must not** provide any form of authorization
to the underlying resources, and authorization must be performed on the request as
with any other regardless of the presence of a page token.
```

Reviewer 2: Thanks, I think that makes it more clear. The main issue I was getting at was that if the page token contains some resource reference, the service should not blindly use that without going through proper checks. Your description makes the advice more direct to that goal.

> to the underlying resources, and authorization must be performed on the request as

Reviewer: ```suggestion
to the underlying resources, and authorization **must be** performed on the request as
```

> with any other regardless of the presence of a page token.

Reviewer: ```suggestion
with any other regardless of the presence of a page token.

```

> with any other regardless of the presence of a page token.
> ### Expiring page tokens

Author: ```suggestion
with any other regardless of the presence of a page token.

### Expiring page tokens
```

> to the underlying resources, and authorization must be performed on the request as
> with any other regardless of the presence of a page token.
> ### Expiring page tokens

Author: ```suggestion
to the underlying resources, and authorization **must** be performed on the request
as with any other regardless of the presence of a page token.

### Expiring page tokens
```
